var/spack/repos/builtin/packages/openfoam/package.py

```diff
@@ -0,0 +1,358 @@
+##############################################################################
+# Copyright (c) 2017 OpenCFD Ltd.
+#
+# This file is released as part of Spack.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for the LLNL notice and the LGPL.
+#
+# License
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+from spack.environment import *
+
+import multiprocessing
+import glob
+import os
+import re
+import shutil
+
+
+# Notes.
+# - Resolution of flex, zlib needs more attention
+# - Paraview/VTK build - needs adjustments in ~/.spack/packages.yaml
+class Openfoam(Package):
```

Comment:
Just waiting for the paraview pull request to get merged and then I can fire off an update for openfoam (since it depends on paraview includes which were previously missing).